crawlerr/server.py

```
@@ -283,45 +309,55 @@ async def get_server_info(ctx: Context) -> Dict[str, Any]:
             collection_created = await vector_service.ensure_collection()
-            logger.info(f"Vector service health: {'✓' if vector_healthy else '✗'}")
-            logger.info(f"Vector collection ready: {'✓' if collection_created else '✗'}")
+            v_status = "[green]✅[/green]" if vector_healthy else "[red]❌[/red]"
+            c_status = "[green]✅[/green]" if collection_created else "[red]❌[/red]"
+            logger.info(f"🗄️  Vector service health: {v_status}")
+            logger.info(f"📚 Vector collection ready: {c_status}")
         
-        logger.info("Crawlerr server started successfully ✓")
+        logger.info("[bold green]🎉 Crawlerr server started successfully![/bold green]")
         
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)